### DISCUSSION TITLE
Add comment to every network within Access Lists

### DISCUSSION THREAD

**Discussion by lodgmar** [NginxProxyManager/nginx-proxy-manager#3774] [Ideas]
I'd really love it. For example:
```
    allow | 8.8.8.8 | Google DNS
    allow | 1.1.1.1 | Cloudfare DNS
    deny | all
```
Of course, these two are well-known IPs, and nobody needs a comment for them, but when you're adding public addresses of your own servers or company routers... it would be very helpful being able to comment every IP, for maintaining reasons.

**Comment by Realmagnum** [comment]
This very useful and expected option!
